test/functional/feature_sysperms.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Verify -sysperms=false (default) works as expected."""
+
+import os
+import platform
+import stat
+
+from test_framework.test_framework import BitcoinTestFramework, SkipTest
+
+
+class SyspermsTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 2
+
+    def check_nosysperms_pathname(self, fullpathname):
+        self.log.info('{}  {}'.format(stat.filemode(os.stat(fullpathname).st_mode), fullpathname))
+        return (os.stat(fullpathname).st_mode & (stat.S_IRWXO | stat.S_IRWXG)) == 0
+
+    def check_nosysperms_dir(self, regtest_dirpath):
+        result = []
+        for path, dirs, files in os.walk(regtest_dirpath):
+            for dirname in dirs:
+                result.append(self.check_nosysperms_pathname(os.path.join(path, dirname)))
+
+            for filename in files:
+                result.append(self.check_nosysperms_pathname(os.path.join(path, filename)))
+        return result
+
+    def run_test(self):
+        self.log.info("Check for valid platform")
+        if platform.system() == 'Windows':
+            raise SkipTest("This test does not run on Windows.")
+
+        self.restart_node(0)
+        datadir = self.nodes[0].datadir
+
+        self.log.info("Running test without -sysperms option - assuming umask of '0o77'")
```

Comment:
`'0o77'` - typo?

Reply:
you're right, should be '077'. What did you make of -sysperm option, is it still needed?

Reply:
> What did you make of -sysperm option, is it still needed?

My concerns about `-sysperms` are in the note here #15902.

Reply:
does it makes sense to close this PR - perhaps replace it with simply removing the option? i suppose the tests might still be useful...